CHANGELOG

```diff
@@ -1,5 +1,11 @@
 Verificarlo CHANGELOG
 
+[v0.2.0] 2016/12/09
+    * Support for LLVM up to 3.8
+    * Support for standalone installation with make install
```

Comment:
For make install in sudo, did you check that clean/distclean and directory removal are still possible for the user, I had some 'root' file in my user directory